### PR TITLE
Changelog and secpoll for upcoming rec-4.2.0 release

### PR DESCRIPTION
**Pull request by omoerbeek**
### Short description

Plus a typo in the rc2 changelog.

### Checklist

I have:
- [X] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [ ] compiled this code
- [ ] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)
- [ ]  checked that this code was merged to master

### CODE REVIEW

File: docs/secpoll.zone [hunk 1]
```diff
@@ -192,6 +192,7 @@ recursor-4.2.0-alpha1.security-status                   60 IN TXT "1 OK"
 recursor-4.2.0-beta1.security-status                    60 IN TXT "1 OK"
```
Comment (pieterlexis) [inline docs/secpoll.zone]: Please change the pre-releases to "2 Unsupported pre-release, no known vulnerabilities"

File: docs/secpoll.zone [hunk 2]
```diff
@@ -188,10 +188,14 @@ recursor-4.1.11.security-status                         60 IN TXT "1 OK"
...
-recursor-4.2.0-beta1.security-status                    60 IN TXT "1 OK"
-recursor-4.2.0-rc1.security-status                      60 IN TXT "1 OK"
-recursor-4.2.0-rc2.security-status                      60 IN TXT "1 OK"
+recursor-4.2.0-alpha1.security-status                   60 IN TXT "2 Unsupported pre-release (no known vulnerabilities)"
+recursor-4.2.0-beta1.security-status                    60 IN TXT "2 Unsupported pre-release
+(no known vulnerabilities)"
+recursor-4.2.0-rc1.security-status                      60 IN TXT "2 Unsupported pre-release
+(no known vulnerabilities)"
+recursor-4.2.0-rc2.security-status                      60 IN TXT "2 Unsupported pre-release
+(no known vulnerabilities)"
```
Comment (mnordhoff) [inline docs/secpoll.zone]: Is it a problem that those are wrapped to two lines?

Comment (Habbie) [reply docs/secpoll.zone]: quite likely yes

Comment (omoerbeek) [reply docs/secpoll.zone]: thanks for spotting

**Pull request merged**